lib/galaxy/datatypes/test/test.gpr

```diff
@@ -0,0 +1,13 @@
+ATF       1.0
+8         5
+"Type=GenePix ArrayList V1.0"

```

Comment:
It looks like this is a GAL file, not GPR.

Reply:
GPR and GAL are quite the same, as well as ATF.

Reply:
For submission to microarrays databases like geo they use gpr format (http://mdc.custhelp.com/app/answers/detail/a_id/18883/kw/18883/session/L2F2LzEvdGltZS8xNTUxMjY5NTc2L3NpZC9MajhQX3E4bw%3D%3D#gpr). I quote from the link "ATF is a tab-delimited text file format that can be read by typical spreadsheet programs such as Microsoft Excel. It is used for GenePix Array List (GAL) files, and GenePix Results (GPR) files.".

Reply:
As far as I understanda, GPR and GAL are "subclasses" of ATF, but are not equivalent. For example, `BlockCount` is a header only for GAL, not GPR.

Reply:
You are right. The test file is there just for sniffing the header for the type. ATF is a format used to describe both GPR and GAL. The most used one is GPR.

Reply:
Got it, I will close this pull, add your suggested modifications and pull back.
Thank you very much